CONTRIBUTING.md

```diff
@@ -112,15 +111,19 @@ and use the following VSCode user settings:
 "editor.formatOnSave": true,
 "rust-analyzer.rustfmt.extraArgs": ["+nightly"],
 "rust-analyzer.check.overrideCommand": [
```

Comment:
on second thought, isn't clippy too heavy a computation if save `onFocusChange` is configured, which is a common setting. wdyt?